slippyMap/resources/LeafletMap.js

```diff
@@ -238,7 +236,12 @@ export default class LeafletMap {
         center: 'flyTo'
       };
     }
-    if (this.featureGroup) {
+    const featureGroupBounds = this.featureGroup.getBounds();
+    let featureGroupCenter;
+    if (featureGroupBounds && featureGroupBounds.isValid()) {
+      featureGroupCenter = featureGroupBounds.getCenter();
+    }
+    if (this.featureGroup && featureGroupBounds && featureGroupCenter) {
```

Comment:
check for featureGroup is not needed, isn't it? it always is at least an empty featureGroup object.